package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha --require @babel/register src/tests/*.js"
```

Comment:
I think before the scripts start executing you should change the environment, may add NODE_ENV=test. so as during testing it changes the environment

Reply:
@gildniy

Reply:
You're right! Thanks

Reply:
> I think before the scripts start executing you should change the environment, may add NODE_ENV=test. so as during testing it changes the environment

You're right, I implemented that.